### PR TITLE
BASIC-893: Added a view of upcoming events at /events

### PR DESCRIPTION
**Pull request by cjwest**

### CODE REVIEW

File: modules/stanford_events_views/stanford_events_views.info [hunk 1]
```diff
@@ -7,8 +7,6 @@ project = stanford_events_views
 dependencies[] = calendar
 dependencies[] = stanford_date_formats
 dependencies[] = stanford_events_importer
-dependencies[] = stanford_image_styles
-dependencies[] = views
```
Comment (cjwest) [inline modules/stanford_events_views/stanford_events_views.info:11]: Do we want these dependencies?

Comment (sherakama) [inline modules/stanford_events_views/stanford_events_views.info:11]: Yes, we want both of those. Please restore them.

File: modules/stanford_events_views/stanford_events_views.info [hunk 2]
```diff
@@ -16,4 +14,6 @@ features[features_api][] = api:2
 features[views_view][] = stanford_events_calendar
 features[views_view][] = stanford_events_manage
 features[views_view][] = stanford_events_views
+features_exclude[dependencies][date_views] = date_views
+features_exclude[dependencies][views] = views
```
Comment (sherakama) [inline modules/stanford_events_views/stanford_events_views.info]: Please remove the 
features_exclude[dependencies][views] = views 
line

**Comment by cjwest** [comment]
Okay, @sherakama, I think it is ready to merge. What do you think?

**Pull request merged**